test/functional/Java8andUp/src_110_up/org/openj9/test/annotation/ContainsRuntimeAnnotationTest.java

```diff
@@ -0,0 +1,188 @@
+ *
+ * This Source Code may also be made available under the following
+ * Secondary Licenses when the conditions for such availability set
+ * forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+ * General Public License, version 2 with the GNU Classpath
+ * Exception [1] and GNU General Public License, version 2 with the
+ * OpenJDK Assembly Exception [2].
+ *
+ * [1] https://www.gnu.org/software/classpath/license.html
+ * [2] http://openjdk.java.net/legal/assembly-exception.html
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+ *******************************************************************************/
+
+import org.testng.Assert;
+import org.testng.AssertJUnit;
+import org.testng.annotations.Test;
+import org.testng.log4testng.Logger;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+import java.lang.annotation.Annotation;
+import java.lang.annotation.ElementType;
+
+import java.util.Arrays;
+
+import jdk.internal.misc.SharedSecrets;
+import jdk.internal.reflect.ConstantPool;
+import jdk.internal.vm.annotation.Stable;
+
+@Test(groups = { "level.sanity" })
+public class ContainsRuntimeAnnotationTest {
+	private static final String THIS_CLASS_NAME = "org/openj9/test/annotation/ContainsRuntimeAnnotationTest"; //$NON-NLS-1$
+	private final ConstantPool constantPool;
+
+	static {
+		try {
+			System.loadLibrary("anntests"); //$NON-NLS-1$
+		} catch (UnsatisfiedLinkError e) {
+			Assert.fail(e.getMessage() + "\nlibrary path = " + System.getProperty("java.library.path")); //$NON-NLS-1$ //$NON-NLS-2$
+		}
+	}
+
+	@MyFieldAnnotation
+	int myField = 0;
+
+	@MyMethodAnnotation
+	void myMethod() {}
+	
+	int myField2 = 0;
+	
+	@MyFieldAnnotation
+	@MyFieldAnnotation2
+	@Stable
+	@MyFieldAnnotation3
+	@MyFieldAnnotation4
+	int myField3 = 0;
+
+	void myMethod2() {}
+
+	public ContainsRuntimeAnnotationTest() {
+		constantPool = SharedSecrets.getJavaLangAccess().getConstantPool(this.getClass());
+	}
+
+	private static native boolean containsRuntimeAnnotation(int cpIndex, String annotationName, boolean isField);
+
+	@Test
+	public void test_field_annotation() throws Exception {
+		boolean annotationFound = false;
+		int cpIndex = getMemberCPIndex("myField", "I", true); //$NON-NLS-1$ //$NON-NLS-2$
+
+		Assert.assertTrue(-1 != cpIndex, "could fine myField");
+		
+		final String annotationName = "Lorg/openj9/test/annotation/MyFieldAnnotation;"; //$NON-NLS-1$
+		final String stableAnnotation = "Ljdk/internal/vm/annotation/Stable;"; //$NON-NLS-1$
+		annotationFound = containsRuntimeAnnotation(cpIndex, annotationName, true);
+		
+		Assert.assertTrue(annotationFound, "did not detect annotation");
+		
+		cpIndex = getMemberCPIndex("myField2", "I", true); //$NON-NLS-1$ //$NON-NLS-2$
+
+		Assert.assertTrue(-1 != cpIndex, "could fine myField2");
```

Comment:
The test contains a lot of empty lines, which can be removed. Example:

```suggestion
		cpIndex = getMemberCPIndex("myField2", "I", true); //$NON-NLS-1$ //$NON-NLS-2$
		Assert.assertTrue(-1 != cpIndex, "could fine myField2");
```